include/wlr/types/wlr_xdg_shell.h

```diff
@@ -191,14 +191,11 @@ struct wlr_xdg_surface {
 	struct wl_list popups; // wlr_xdg_popup::link
 
 	bool added, configured, mapped;
-	uint32_t configure_serial;
 	struct wl_event_source *configure_idle;
 	uint32_t configure_next_serial;
```

Comment:
I'd rename this to `scheduled_serial` to follow xdg-toplevel/layer-surface fields naming. (Although this is maybe slightly out of scope of this PR.)

Reply:
Done